modules/appeals_api/app/swagger/appeals_status/v1/api_description.md

```diff
@@ -11,7 +11,7 @@ To file an appeal or decision review, use one of these APIs:
 
 ## Background
 
-The Appeals API passes data through to Caseflow, a case management system. Caseflow accepts a header with the Veteran’s SSN and returns the current status of their decision reviews and/or benefits appeals.
+The Appeals API passes data through to Caseflow, a case management system. Caseflow accepts a header with the Veteran’s ICN and returns the current status of their decision reviews and/or benefits appeals.
```

Comment:
While it probably doesn't matter that much (since the end user doesn't actually see the header sent to Caseflow), I think this could change to just say that we send the Veteran's data to Caseflow (or something along those lines). We're accepting the ICN from our users now in v1, but we're still using that ICN to look up the SSN and send that SSN to Caseflow, so we don't actually send the ICN to Caseflow at all.

Reply:
Oh, good call! I don't think I read that sentence closely enough. I thought it said "The Appeals Status API accepts a header with the Veteran's SSN."

What do you think about the following wording to make it more generic:
`The Appeals Status API passes data through to Caseflow, a case management system. Caseflow returns the current status of their decision reviews and/or benefits appeals.`

My thinking behind removing the mention of SSN is that it might confuse the user when they know they aren't passing an SSN. We could explain that we look up the SSN from the ICN, but that sounds like an implementation detail that the consumer wouldn't care about. What do you think?

Reply:
Michael H. and Melinda C. want to make the proposed wording change above, so I'm going to go ahead and update it.

Reply:
Updated! This is ready for re-review.